README.md

```diff
@@ -89,9 +89,17 @@ const Block = require('ipfs-block')
 
 ### Block
 
-#### `new Block(data, [type])`
+#### `new Block(data, key, [type])`
 
-Creates a new block with raw data `data`. `type` can be either `'protobuf'` or `'ipld'`
+- `data: Buffer|String`
+- `key: Buffer` the multihash of the data
+- `type: String` optional. Defaults to `'protobuf'`
+
+Creates a new block with raw data `data`. `type` can be either `'protobuf'` or `'ipld'`.
+
+#### `Block.create(data, key, [type, ] callback)`
```

Comment:
Let's just go ahead and take from https://github.com/ipfs/js-ipfs-block/pull/5 the fact that .key becomes a function and make it an async function.